docs/ref/utils.txt

```diff
@@ -700,6 +700,12 @@ escaping HTML.
     A version of Python's :func:`urllib.parse.urlencode` function that can
     operate on ``MultiValueDict`` and non-string values.
 
+    .. versionchanged:: 2.2
```

Comment:
Generally, we don't include versionchanged notes where no surrounding text is changed. If you don't think any text should remain after the annotation would be removed, then don't include the annotation. My opinion is that the annotation isn't required.